src/thermo/critProperties.xml

```diff
@@ -0,0 +1,6331 @@
+           <Zc value="0.224"> </Zc>
+           <omega value="0.31"> </omega>
+       </species>
+       <species name= "CH3NO2">
+           <ChemName name ="methyl-nitrite"> </ChemName>
+           <MW value = "61.04"> </MW>
+           <Tc value= "0"> </Tc>
+           <Pc value= "0E6 "> </Pc>
+           <Vc value="0"> </Vc>
+           <Zc value="0"> </Zc>
+           <omega value="0"> </omega>
+       </species>
+       <species name= "CH3NO3">
+           <ChemName name ="methyl-nitrate"> </ChemName>
+           <MW value = "77.04"> </MW>
+           <Tc value= "0"> </Tc>
+           <Pc value= "0E6 "> </Pc>
+           <Vc value="0"> </Vc>
+           <Zc value="0"> </Zc>
+           <omega value="0"> </omega>
+       </species>
+       <species name= "CH4">
+           <ChemName name ="methane"> </ChemName>
+           <MW value = "16.043"> </MW>
+           <Tc value= "190.7"> </Tc>
+           <Pc value= "46.3E6 "> </Pc>
+           <Vc value="98.9"> </Vc>
+           <Zc value="0.288"> </Zc>
+           <omega value="0.011"> </omega>
+       </species>
+       <species name= "CH4O">
+           <ChemName name ="methanol"> </ChemName>
+           <MW value = "32.042"> </MW>
+           <Tc value= "513"> </Tc>
+           <Pc value= "80.1E6 "> </Pc>
+           <Vc value="117.9"> </Vc>
+           <Zc value="0.224"> </Zc>
+           <omega value="0.556"> </omega>
+       </species>
+       <species name= "CH4S">
+           <ChemName name ="methanethiol"> </ChemName>
+           <MW value = "48.103"> </MW>
+           <Tc value= "469.7"> </Tc>
+           <Pc value= "71.9E6 "> </Pc>
+           <Vc value="146.4"> </Vc>
+           <Zc value="0.268"> </Zc>
+           <omega value="0.153"> </omega>
+       </species>
+       <species name= "CH5N">
+           <ChemName name ="methylamine"> </ChemName>
+           <MW value = "31.057"> </MW>
+           <Tc value= "429.6"> </Tc>
+           <Pc value= "74.4E6 "> </Pc>
+           <Vc value="120.5"> </Vc>
+           <Zc value="0.213"> </Zc>
+           <omega value="0.292"> </omega>
+       </species>
+       <species name= "C2CL4">
+           <ChemName name ="tetrachloroethene"> </ChemName>
+           <MW value = "165.834"> </MW>
+           <Tc value= "620.2"> </Tc>
+           <Pc value= "47.6E6 "> </Pc>
+           <Vc value="289.6"> </Vc>
+           <Zc value="0.25"> </Zc>
+           <omega value="0.25"> </omega>
+       </species>
+       <species name= "C2CL6">
+           <ChemName name ="hexachloroethane"> </ChemName>
+           <MW value = "236.74"> </MW>
+           <Tc value= "704.35"> </Tc>
+           <Pc value= "39.36E6 "> </Pc>
+           <Vc value="419.5"> </Vc>
+           <Zc value="0.282"> </Zc>
+           <omega value="0.275"> </omega>
+       </species>
+       <species name= "C2CL3F">
+           <ChemName name ="3  112trichlorotrifluoroet"> </ChemName>
+           <MW value = "hane 187.376"> </MW>
+           <Tc value= "487.3"> </Tc>
```

Comment:
How is this file being generated? There is clearly a parsing error here -- the `3` at the start of `ChemName` should be the end of the `species` name, and the `hane` at the start of the molecular weight should be at the end of the `ChemName`. There are over 100 species with errors of this sort in the file.